packages/core/src/internal/base/focus-trap.base.ts

```diff
@@ -21,6 +25,16 @@ export class CdsBaseFocusTrap extends LitElement {
   @internalProperty({ type: Boolean, reflect: true })
   protected __demoMode = false;
 
+  @property({ type: String }) focusTrapId: string;
+
+  constructor() {
+    super();
+    this.focusTrapId = createId();
+    // if we see issues instantiating here, we should consider moving to
```

Comment:
yep. if we start building out components and see them fail in the react wrappers and such. it would have to be an async issue. doesn't seem to be a problem right now. but i've seen it before.